File: atividade haskell/temperatura.hs
main :: IO ()
main = do
    putStrLn "Digite a temperatura em Celsius:"
    input <- getLine
    let temperaturaCelsius = read input :: Double

    let temperaturaFahrenheit = celsiusParaFahrenheit temperaturaCelsius

    putStrLn $ "A temperatura em Fahrenheit é: " ++ show temperaturaFahrenheit

celsiusParaFahrenheit :: Double -> Double
celsiusParaFahrenheit celsius = (celsius * 1.8) + 32

File: atividade c/temperatura.c
#include <stdio.h>

int main() {
    double celsius, fahrenheit;

    printf("Digite a temperatura em Celsius: ");
    scanf("%lf", &celsius);

    fahrenheit = (celsius * 1.8) + 32;

    printf("A temperatura em Fahrenheit é: %.2lf\n", fahrenheit);

    return 0;
}